operations/patch_util.go

```diff
@@ -461,7 +508,7 @@ func getAPIPatchDisplay(apiPatch *restmodel.APIPatch, summarize bool, uiHost str
 		return "", errors.New("service patch is not a Patch")
 	}
 
-	return getPatchDisplay(&servicePatch, summarize, uiHost)
+	return getCommitQueuePatchDisplay(&servicePatch, summarize, uiHost)
```

Comment:
is getAPIPatchDisplay only used for commit queue, in which case should it also be called getAPICommitQueuePatchDisplay?

Reply:
Yes, we can rename to that clearer function name